src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/Instance.cs

```diff
@@ -93,8 +87,8 @@ public override string ToString()
         /// <summary>
         /// Sets platform self links for the instance.
         /// </summary>
-        /// <param name="storageHostAndBase">The host and basepath for platform storage. E.g. 'at22.altinn.cloud/storage/api/v1/'. Must end with '/'.</param>
-        public void SetPlatformSelflink(string storageHostAndBase)
+        /// <param name="storageHostAndBase">The host and base path for platform storage. E.g. 'at22.altinn.cloud/storage/api/v1/'. Must end with '/'.</param>
+        public void SetPlatformSelfLinks(string storageHostAndBase)
         {
```

Comment:
@SandGrainOne Is this method never used anywhere since it's possible to rename it? Or is this a breaking change that should've been a new major version?

Reply:
@altinnadmin This should not be used by anyone except Storage and it would surprise me a lot if anyone does, but I can only check our own code.

Reply:
:+1: